docs/Configuration.md

```diff
@@ -0,0 +1,49 @@
+For classes that support IConfiguration based configuration:

```

Comment:
I'd prefer this be embedded in the [SecureMemory C# Edition docs](/godaddy/asherah/tree/master/csharp/SecureMemory/README.md) for the time being. We can revisit as we look to port these features to the Java and Go SDKs

Reply:
That does make more sense